core/code/boot.js

```diff
@@ -686,43 +686,93 @@ function prepPluginsToLoad() {
   // executes setup function of plugin
   // and collects info for About IITC
   function safeSetup (setup) {
-    if (!setup) {
```

Comment:
1. You should avoid mixing main changes with whitespace changes
2. https://github.com/IITC-CE/ingress-intel-total-conversion/wiki/Code-Style